librz/main/rizin.c

```diff
@@ -911,7 +909,7 @@ RZ_API int rz_main_rizin(int argc, const char **argv) {
 			LISTS_FREE();
 			return 1;
 		}
-	} else if ((argc >= 2 && strcmp(argv[opt.ind - 1], "--")) || ((!strcmp(argv[opt.ind - 1], "--") && argv[opt.ind]))) {
+	} else if ((argc >= 2 && argv[opt.ind] && strcmp(argv[opt.ind], "--")) || ((!strcmp(argv[opt.ind - 1], "--") && argv[opt.ind]))) {
```

Comment:
Can you extract this whole if in a small inline function?

Reply:
wait, didn't we change `-` to `=` ?

Reply:
yes we did. This is `--`.

For example, `rizin -- -test`, `rizin -d -- /bin/ls -l /`.